### PR TITLE
:sparkles: (nordigen) release the feature

### PR DESCRIPTION
**Pull request by MatissJanis**
All the major action items of https://github.com/actualbudget/actual/issues/724 are now complete. I've been actively using and testing the sync for the last few months and IMO it's good to get released.

There will be some further action-items to rename things from Nordigen to GoCardLess, but not yet. There is no clear migration path for it as of right now.

As for further bug reports and feature requests - I'll close off the main ticket and allow folks to create individual issues for anything new that comes up.

### CODE REVIEW

**Review by j-f1 — COMMENTED** [review]
Some things that I think should be changed before merging:

- [ ] the “Add Account” modal should not prioritize syncing with Noridgen, especially due to the privacy concerns and region restrictions. We don’t have anywhere that I know of that has two primary buttons, so I would recommend styling the Nordigen button as a regular non-primary button, just like the “Create local account” button. (I can open a PR to do this shortly)
- [ ] Errors in the “Set-up Nordigen” process are not surfaced to the user.
- [ ] “Nordigen integration is only available for budgets using actual-server. [Learn more.](https://actualbudget.github.io/docs/Installing/overview)” should link to the bank syncing docs (which need to be updated to reflect their out-of-experimental status)

(sorry that I didn’t surface these earlier)

**Comment by MatissJanis** [comment]
Good ideas, thanks Jed!

**Comment by Jackenmen** [comment]
I think that error handling for 429s (rate limit issues presented as non-descriptive "Failed syncing account “...”" in the UI) and 503s (presented as Internal Error in the UI, unsure if that one hasn't already been fixed tho) should be done before the feature is released but it doesn't seem like anyone was interested in working on this for the last few months that this feature was available so it might be hard to get someone to do it 😄

File: packages/desktop-client/src/hooks/useFeatureFlag.ts [hunk 1]
```diff
@@ -2,7 +2,6 @@ import { useSelector } from 'react-redux';
 
 const DEFAULT_FEATURE_FLAG_STATE: Record<string, boolean> = {
   reportBudget: false,
-  syncAccount: false,
```
[the remark this answers is not in the document]
Comment (MatissJanis) [reply packages/desktop-client/src/hooks/useFeatureFlag.ts:5]: Don't think it's worth the effort tbh.

**Comment by MatissJanis** [comment]
Ok, out for review again.

All should be done except extra error handling. IMO if nobody has reported this in the last few months - it should not hold back the feature release.

We can circle back if/when issues are reported again.

**Comment by Jackenmen** [comment]
> IMO if nobody has reported this in the last few months - it should not hold back the feature release.

Note that it was reported at the very beginning: https://github.com/actualbudget/actual/issues/724#issuecomment-1455595011
So it may be that people saw that this was already reported and simply didn't choose to report the same thing again but I guess it's hard to tell how important this is.

**Review by j-f1 — APPROVED** [review]
I’m ok with releasing this now and iterating on it more later as needed.

**Pull request merged**